### ISSUE TITLE
After some time Listening works but texttospeech not any more

### ISSUE THREAD

**Issue by AleksCee** [586837r/node-red-contrib-alexa-remote2#155]
Hi,

I use the node since 2 weeks. But now the routine speak node only works after redeploy and after some time it‘s stop working.
The listening node seams to work because other nodes in the flow are triggered. The speak node seams trigger too (the status change zu success) but nothing happens. After a redeploy, it works again.

I use the proxy mode with a auth file. No errors in the debug.

Any ideas? thanks Alex

**Comment by AleksCee** [comment]
It’s seams that the refresh dose not the same as a deployment? Is there e way to restart/initial the node instead of a refresh? The redeploy seams to fix the connection.

**Comment by RogerHill** [comment]
I am having this issue almost daily. I currently recover by making a slight change to the flow and re-deploying.

**Comment by AleksCee** [comment]
I think I found a solution.

![image](https://user-images.githubusercontent.com/2960655/93663025-ebebe380-fa64-11ea-85bb-1996858b62ea.jpeg)

Every hour a cookie refresh and if this fail a reauthorization with the catch node.

Alex

**Comment by ammawel** [comment]
I have the same error, but I don't think a missing or expired cookie is the reason, because all other alexa-functions are working fine.
The debug window also shows the correct speak-payload - there is just no output and Alexa does not say it.
Achim

**Comment by ammawel** [comment]
it seems to be a known problem - still without a permanently working solution

[Alexa forgets how to talk #99](https://github.com/586837r/node-red-contrib-alexa-remote2/issues/99)
[Speak and other commands stop working after a few hours #40](https://github.com/586837r/node-red-contrib-alexa-remote2/issues/40)

Achim

**Comment by jirmjos** [comment]
I suffer same issue. Once redeploy flow work for some time, but after a day or so it stops to send any to my Echo device.
I'm not debugging what's happened but visually all nodes are ready and not raising error.
I'm ussing v3.10.4 (I think latest) but this issue is really anoying.

Any news about how to solve it ?

**Comment by ammawel** [comment]
No, unfortunately there is nothing new. I'm not sure if the module will get any updates, because there is no contributor activity since the end of July.
I use a comparable module in FHEM, which runs absolutely stable with only one refresh a day. I will probably not use Alexa with node-red anymore.
